ArduCopter/Parameters.h

```diff
@@ -47,6 +47,7 @@ class Parameters {
         k_param_g2, // 2nd block of parameters
         k_param_NavEKF3,
         k_param_BoardConfig_CAN,
+        k_param_osd,
```

Comment:
move to g2 please on all platforms

Reply:
no, it needs to be in the top level as it is 3 levels deep, so it won't work in g2